### ISSUE TITLE
There should be some sort of Banner warning when a new update came?

### ISSUE THREAD

**Issue by Lucas559-noob** [younesaassila/ttv-lol-pro#129]
On 1.9, it was implemented a banner to show when a new update released, right? But nothing show, not in the interface, neither on config

> Added "New update available" notification banner in popup for users of unpacked version (Fixes https://github.com/younesaassila/ttv-lol-pro/issues/82)

Also how should it show?

**Comment by EldritchEevee** [comment]
![image](https://github.com/younesaassila/ttv-lol-pro/assets/112034382/623e8666-3958-4308-b79c-ee08e7106703)
Check this. If you're on Firefox, there is currently no update (see the [1.9.1](https://github.com/younesaassila/ttv-lol-pro/releases/tag/v1.9.1) patch notes for more info as to why.)
personal note: direct image uploading is really, **really** good.
tl;dr: XPI file is **still** not signed by Mozilla.

**Comment by younesaassila** [comment]
This was indeed implemented, thanks @EldritchEevee for the screenshot above. It is disabled by default if you have installed the version with automatic updates.
You might not see it on Firefox because the latest version still hasn't been signed. The Mozilla Addons team got back to me yesterday that they got different output than what I uploaded. Turns out I forgot to disable source maps in prod build which can be different across build environments. Fixed it and now waiting for their response.
The only change in that update is just that "Reset player on midroll" is now disabled by default. No other change. So no worries to have here.
